mod-orders/examples/expectCollection.sample

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
team, we decided to implement endpoint to change status of multiple pieces to Expected, also field "comment" edited in UI as well, so we included it in schema. Contract is the same as other collectyion like checkingCollection and receiveCollection